make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -138,24 +133,17 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBAWT, \
     OPTIMIZATION := HIGHEST, \
     CFLAGS := $(CFLAGS_JDKLIB) $(LIBAWT_CFLAGS), \
     EXTRA_HEADER_DIRS := $(LIBAWT_EXTRA_HEADER_DIRS), \
-    DISABLED_WARNINGS_gcc_awt_ImagingLib.c := unused-function sign-compare, \
     DISABLED_WARNINGS_gcc_awt_LoadLibrary.c := unused-result, \
-    DISABLED_WARNINGS_gcc_awt_Mlib.c := unused-function, \
-    DISABLED_WARNINGS_gcc_awt_parseImage.c := sign-compare unused-function, \
-    DISABLED_WARNINGS_gcc_debug_trace.c := unused-function, \
     DISABLED_WARNINGS_gcc_ProcessPath.c := maybe-uninitialized, \
     DISABLED_WARNINGS_gcc_Region.c := maybe-uninitialized, \
     DISABLED_WARNINGS_gcc_SurfaceData.c := unused-value, \
     DISABLED_WARNINGS_gcc_TransformHelper.c := sign-compare, \
-    DISABLED_WARNINGS_clang_awt_ImagingLib.c := sign-compare deprecated-non-prototype, \
-    DISABLED_WARNINGS_clang_awt_parseImage.c := sign-compare, \
-    DISABLED_WARNINGS_clang_debug_mem.c := extern-initializer format-nonliteral, \
+    DISABLED_WARNINGS_clang_awt_ImagingLib.c := deprecated-non-prototype, \
```

Comment:
I see you declared the functions as `inline`, it was probably the intention for declaring them in the header file.

Did it not resolve `deprecated-non-prototype` warning? If not, did you submit another bug to address it?

Reply:
Thanks for reviewing!

I didn't see the warning in mach5 build. However, the warning was not related to inline declarations. See [JDK-8300169](https://bugs.openjdk.org/browse/JDK-8300169) where the warning was reported; the compiler complained that `sMlibFns.fptr` does not have a prototype. See the declaration:
https://github.com/openjdk/jdk/blob/0deb648985b018653ccdaf193dc13b3cf21c088a/src/java.desktop/share/native/libawt/awt/medialib/awt_ImagingLib.h#L35
I'm pretty sure that given the right compiler, the warning would still appear.

After I filed the ticket for this change, I found [JDK-8074823](https://bugs.openjdk.org/browse/JDK-8074823); let's use that to keep track of the remaining warnings.